.github/workflows/continuous-build.yml

```diff
@@ -59,6 +52,38 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
 
+      - name: Download package signing GPG secret key
+        if: ${{ runner.os == 'Linux' && github.event_name == 'push' }}
+        run: |
+          aws secretsmanager get-secret-value --region us-east-1 --secret-id "XRayDaemonSecrets" | jq -r ".SecretString" > aws-xray-secret.gpg
+          md5sum aws-xray-secret.gpg
+
+      - name: Import signing GPG key
+        if: ${{ runner.os == 'Linux' && github.event_name == 'push' }}
+        run: |
+          gpg --import aws-xray-secret.gpg
+          gpg --list-keys
+          gpg --armor --export -a "aws-xray@amazon.com" > aws-xray-public.gpg
+          rpm --import aws-xray-public.gpg
+          shred -fuvz aws-xray-secret.gpg
```

Comment:
probably good to remove public key using `shred` as well.

Reply:
I don't think it is a problem since public key can be shared or kept around. But I agree that we should shred it as well.